lib/inferno_template.rb

```diff
@@ -2,20 +2,38 @@
 
 module InfernoTemplate
   class Suite < Inferno::TestSuite
-    id :test_suite_template
-    title 'Inferno Test Suite Template'
-    description 'A basic test suite template for Inferno'
+    id :inferno_template_test_suite
+    title 'Inferno Template Test Suite'
+    description 'Inferno template test suite.'
+
+    # These inputs will be available to all tests in this suite
+    input :url,
+          title: 'FHIR Server Base Url'
 
-    # This input will be available to all tests in this suite
-    input :url
     input :access_token
 
+    input :credentials,
+          title: 'OAuth Credentials',
+          type: :oauth_credentials,
+          optional: true
+
```

Comment:
I think we want to remove this input and just have the access_token input instead. Or change the tutorial to say to add a new Oauth Credentials input